net.sf.eclipsecs.checkstyle/META-INF/MANIFEST.MF

```diff
@@ -7,10 +7,10 @@ Bundle-Vendor: Eclipse Checkstyle Project
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Export-Package: .,
- com.google.common.base;version="10.5.0",
- com.google.common.cache;version="10.5.0",
- com.google.common.collect;version="10.5.0",
- com.google.common.io;version="10.5.0",
+ com.google.common.base;version="10.6.0",
```

Comment:
The only difference with the PRs I usually make are those version numbers in Export-Package which I leave untouched and are bumped later by the release script.
I do not know what they refer to, though.

Reply:
Those are re-exports of the packages already exported by the checkstyle core library. This makes the libraries easily available in the eclipse-cs code, but there are multiple problems with this approach to my mind:
* the version numbers are plain wrong. Those are Google Guava packages, and guava has very different version numbers for its packages. That can easily lead to problems when other plugins also import the same packages and expect correct version numbers.
* the plugin should not depend on the dependencies of the core library in _its own code_. That way every upgrade of the core library has the potential to break code in the eclipse plugin for no good reason (e.g. if a method in Google Guava gets removed).

I'll create another change to get rid of these exported packages here, and will rebase this change afterwards.

Reply:
@Calixte The exports and the changed versions here are gone now. Thanks for the extra attention on the review and reporting this. Sometimes I'm quite surprised by what unusual things we find in the code here.

Reply:
It makes much more sense now, thank you.